extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java

```diff
@@ -237,7 +242,7 @@ private String importSinglePhoto(
   }
 
   // Replace any non-whitespace, non-letter, or non-digit characters with -
-  static String cleanName(String name) {
+  static String cleanAlbumName(String name) {
```

Comment:
Are the restrictions different for album and photo names? Can we use the same function for both? I wrote this version up as a quick fix for the albums but its not behavior that we rely on.

Reply:
I assumed there was some reason for the cleanName, we can use the same though.